src/App.jsx

```diff
@@ -1,40 +1,33 @@
-import { useState } from "react";
-import TodoInput from "./component/TodoInput.jsx";
-import TodoList from "./component/TodoList.jsx";
+import { useState } from "react"
+import TodoInput from "./component/TodoInput.jsx"
+import TodoList from "./component/TodoList.jsx"
 
 export default function App() {
-  const [todos, setTodos] = useState([]);
-  const [task, setTask] = useState("");
-  const [flag, setflag] = useState(false);
```

Comment:
This state was only used for checking the reset condition, which was replaced with:
todosList.jsx@25 `todos.length > 0`